modules/statics/src/account.ts

```diff
@@ -36,8 +37,8 @@ export class AccountCoin extends BaseCoin {
 
   constructor(options: AccountConstructorOptions) {
     super({
-      ...options,
       kind: CoinKind.CRYPTO,
```

Comment:
since `kind` is now supplied in options I do not think the spread (`...`) is needed (or `kind` here)

Reply:
We still need the spread and the `kind` parameter since in the "AccountConstructorOptions" interface `kind` is optional and in the base class "BaseCoin" it's required.